python/extract_decrypt_kms.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python3
+
+"""
+@attention: inspired by
+- https://github.com/zalando/kmsclient on 2015 10 15
+- https://github.com/zalando-stups/taupage/blob/master/runtime/opt/taupage/bin/decrypt_kms.py on 2015 10 15
+"""
+
+import argparse
+import yaml
+import boto3
+import base64
+# import pprint
+
+
+class Kms:
+
+    @classmethod
+    def __init__(cls,
+                 file: str = None,
+                 key: str = None,
+                 region: str = None):
+        cls.file = file
+        cls.key = key
+        cls.region = region
+        cls.service_name = 'kms'
```

Comment:
Since `service_name` is always `kms` you can assign it as a class property just by putting `service_name = 'kms'` before the `__init__`